app/db.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import create_engine
+
+import os
+
+from sqlalchemy.ext.declarative import declarative_base
+
+user = os.getenv('POSTGRES_USER')
+password = os.environ.get('POSTGRES_PASSWORD')
+database = os.environ.get('POSTGRES_DB')
+
+
+engine = create_engine(f'postgresql://{user}:{password}@localhost:5432/{database}', echo=True)
```

Comment:
there is an environment variable for the host.

Reply:
```py
host = os.getenv('POSTGRES_HOST')
port = os.getenv('POSTGRES_PORT', '5432')

engine = create_engine(f'postgresql://{user}:{password}@{host}:{port}/{database}', echo=True)
```